src/contrib/cluster/Akka.DistributedData/Replicator.cs

```diff
@@ -1051,7 +1052,7 @@ private void ReceiveStatus(IImmutableDictionary<string, ByteString> otherDigests
             var otherKeys = otherDigests.Keys.ToImmutableHashSet();
             var myKeys = (totChunks == 1
                     ? _dataEntries.Keys
-                    : _dataEntries.Keys.Where(x => Math.Abs(MurmurHash.StringHash(x)) % totChunks == chunk))
+                    : _dataEntries.Keys.Where(x => Math.Abs(MurmurHash.StringHash(x) % totChunks) == chunk))
```

Comment:
This was an extremely frustrating issue to find - the `Math.Abs` needed to be called on the results _of the modulus_, not the results of the hash. Otherwise, certain keys ("D8" in this case) would never be included in the gossip reply back for replication.

Reply:
Although I kind of goofed here and changed to things at once: this and the way `IsOtherDifferent` is computed using `ByteString`s for digests.